src/main/scala/org/ergoplatform/settings/Constants.scala

```diff
@@ -44,7 +45,7 @@ object Constants {
   // Number of last block headers available is scripts from ErgoStateContext
   val LastHeadersInContext = 10
 
-  val modifierSerializers: Map[NetworkObjectTypeId.Value, ScorexSerializer[_ <: NodeViewModifier]] =
+  val modifierSerializers: Map[NetworkObjectTypeId.Value, ErgoSerializer[_ <: NodeViewModifier]] =
```

Comment:
Please add ScalaDoc for touched fields.

Reply:
Fixed here, but for initial difficulty and nbits, they are used only in 

```
  val fake: PreHeader = CPreHeader(
    version = 0.toByte,
    parentId = Header.GenesisParentId,
    timestamp = 0,
    nBits = Constants.InitialNBits,
    height = ErgoHistory.EmptyHistoryHeight,
    votes = Array.fill(3)(0.toByte),
    minerPk = org.ergoplatform.mining.group.generator
  )
```

And idk why it is done this way by @greenhat , can you elaborate on this, write ScalaDoc for fake maybe ?

Reply:
Sure. I pushed a ScalaDoc comment for `PreHeader.fake` to this branch.

Reply:
@greenhat thanks